src/go/rpk/pkg/cli/cmd/wasm/generate_test.go

```diff
@@ -20,6 +21,49 @@ func init() {
 	inTests = true
 }
 
+func TestJsonParse(t *testing.T) {
```

Comment:
This test excercises `getWasmApiVersion`, so it should be called `TestGetWasmApiVersion`.

Reply:
Done